### PR TITLE
Ot90 660

### PR DESCRIPTION
**Pull request by juanmanuell88**
https://alkemy-labs.atlassian.net/browse/OT90-684
Delete user endpoint

### CODE REVIEW

File: components/user/network.js [hunk 1]
```diff
@@ -37,4 +37,18 @@ router.post('/',
       })
   })
 
+router.delete('/:id', isAdmin, async (req, res) => {
+  const { params: { id } } = req
+  try {
+    const deleted = await controller.deleteUser(id)
+    if (!deleted) {
+      res.status(404).send({ Error: `Not founded a user with id ${id}` })
```
Comment (benjaminPla) [inline components/user/network.js]: "a user with that id was not found"

**Review by benjaminPla — APPROVED** [review]

File: components/user/network.js (hunk 1 above)
Comment (juanmanuell88) [reply components/user/network.js]: @benjaminPla  Done

File: components/user/network.js [hunk 2]
```diff
@@ -37,4 +37,18 @@ router.post('/',
...
 
+router.delete('/:id', isAdmin, async (req, res) => {
+  const { params: { id } } = req
+  try {
+    const deleted = await controller.deleteUser(id)
+    if (!deleted) {
+      res.status(404).send({ Error: `A user with that ${id} was not found` })
+      return
+    }
+    res.status(204).send()
```
Comment (dvelezroman) [inline components/user/network.js]: copiar formato de response

**Review by dvelezroman — CHANGES_REQUESTED** [review]
fix response formatting

**Review by andresmonsa — APPROVED** [review]

File: components/user/network.js (hunk 2 above)
Comment (juanmanuell88) [reply components/user/network.js]: @dvelezroman  done ... for example
image

**Review by dvelezroman — APPROVED** [review]

**Pull request merged**